app/components/ToggleDisplay.js

```diff
@@ -80,28 +79,17 @@ export default class ToggleDisplay extends Component {
 
   runHelpers = () => {
     if (this.state.helpers) {
```

Comment:
Wow good work. Yeah you're right that the only reason this isn't breaking is that the helpers [always](https://github.com/ChessCom/browser-extension/blob/fb5abf897e5ddacb1c8742e7d7d8bd7e5c144c9b/app/components/Options/Config.js#L282-L303) has a length of just 1. Jiminy. Your refactoring allows for a length of more than this, plus it's much cleaner. Nice. 

> Also, just to make sure I understand what helpers are, they are elements of html that are connected to the element we actually want to hide that are so connected they should also be hidden, yeah?

The helpers aren't themselves DOM elements, just to be clear. Probably the best way to understand what the Helpers are is by looking at [the Config object](https://github.com/ChessCom/browser-extension/blob/fb5abf897e5ddacb1c8742e7d7d8bd7e5c144c9b/app/components/Options/Config.js). I'm not sure `helper` is the most useful name in this context but it's essentially a map that is paired with the user-specified localStorage settings to interact with the DOM.